### PR TITLE
Init blog app and create Post model for it

### PR DESCRIPTION
**Pull request by slsh1o**
Finish "models" part of a djangogirls tutorial and switch to PostgreSQL database.
Also set migrations folder to be ignored by pylint and pydocstyle. This minor changes aren't from tutorial but linked with models thus I decided to leave commit w/ this changes in this PR.

### CODE REVIEW

File: blog/apps.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (webknjaz) [inline blog/apps.py]:
```suggestion
```

Comment (slsh1o) [reply blog/apps.py]: For educational purpose used rebase -i to fix this issue)

Comment (webknjaz) [reply blog/apps.py]: sure. it's often useful to accept a lot of small patches first and then do interactive rebase and apply them as fixups in different places. this can be done after some rounds of review.

File: mysite/settings.py [hunk 2]
```diff
@@ -57,8 +58,13 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.getenv('KPI_DB'),
```
Comment (webknjaz) [inline mysite/settings.py]: This is nice but also it does not belong to this PR. You should submit unrelated changes separately.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Review by serhii73 — DISMISSED** [review]

**Comment by slsh1o** [comment]
I will migrate to Postgres later so redundant commit was removed. Now everything should be fine.

**Comment by webknjaz** [comment]
> Create Post model and makemigrations for it

I'd say that adding that "and blah blah blah" part is unnecessary since migrations are more or less part of the model creation. Also, it makes the commit look like it does more things than it should.

But that's minor, I'll approve it.

**Review by webknjaz — APPROVED** [review]
